src/components/chip/index.tsx

```diff
@@ -127,6 +127,10 @@ export type ChipProps = ViewProps & TouchableOpacityProps & {
    * Dismiss container style
    */
   dismissContainerStyle?: StyleProp<ImageStyle>;
+  /**
+   * Recorder tag
+   */
+  recorderTag?: RecorderTag;
```

Comment:
I don't think you need this because it inherits `TouchableOpacityProps` (remove from API if this is true).